daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2126,9 +2157,11 @@ func (devices *DeviceSet) removeDevice(devname string) error {
 	return err
 }
 
-func (devices *DeviceSet) cancelDeferredRemoval(info *devInfo) error {
+func (devices *DeviceSet) cancelDeferredRemoval(info *devInfo) (bool, error) {
```

Comment:
And I realize this is probably totally subjective, but LMK what you think.

Not a big fan of adding this bool return.